### PR TITLE
Add Gen 8 moves and Pokemon

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

**Comment by AustinXII** [comment]
Looks good. I’ll let you finish what ever you want to edit before merging

File: calc/src/move.ts [hunk 1]
```diff
@@ -58,6 +63,56 @@ export class Move {
     let data: MoveData & {name: string} = extend(true, {name}, MOVES[gen][name], options.overrides);
 
     // If isZMove but there isn't a corresponding z-move, use the original move
+    if (options.useMax && 'maxPower' in data) {
+      const maxMoveName: string = getMaxMoveName(data.type, options.species, !!(data.category === 'Status'));
```
Comment (KrisXV) [inline calc/src/move.ts]: by the way i dont actually know where options.species or options.item would get assigned so this does almost nothing right now

File: calc/src/mechanics/gen8.ts [hunk 2]
```diff
@@ -396,17 +396,17 @@ function calculateModern(
...
+      basePower = move.bp * (['Gmax', 'Dynamax'].indexOf(defender.name) !== -1 ? 2 : 1);
       description.moveBP = basePower;
       break;
     case 'Payback':
       basePower = turnOrder === 'LAST' ? 100 : 50;
       description.moveBP = basePower;
       break;
-    case 'Bolt Break':
+    case 'Bolt Beak':
     case 'Fishious Rend':
```
Comment (KrisXV) [inline calc/src/mechanics/gen8.ts]: also this is throwing an error which is contributing to the failing build

`src/mechanics/gen8.ts:408:5 - error TS7029: Fallthrough case in switch.`

not sure why

Comment (KrisXV) [reply calc/src/mechanics/gen8.ts]: im dumb

File: src/index.template.html [hunk 3]
```diff
@@ -701,6 +701,17 @@
                                 <label class="btn btn-xwide" for="srR">Stealth Rock</label>
                             </div></td>
                         </tr>
+                        <tr class="gen-specific g4 g5 g6 g7 g8">
```
Comment (AustinXII) [inline src/index.template.html]: I don’t think steel spikes was around in gens 4, 5, 6, and 7 O.o

Comment (KrisXV) [reply src/index.template.html]: Done

**Pull request merged**